### PR TITLE
feat: Expose log_analytics variable and create _Default bucket in TF …

### PR DESCRIPTION
**Pull request by Flipez**
…if true

### CODE REVIEW

**Comment by jasonthomas** [comment]
Since the logging bucket config is already provisioned automatically for projects for _Default and _Required buckets, (as per https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/logging_project_bucket_config), in your testing, does this require that the existing resource to be imported into state?

**Comment by jasonthomas** [comment]
If the resource can just be created without an import (that is, we don't need to reconcile any config differences), I'd say we should just create `google_logging_project_bucket_config` resource without the condition on if `log_analytics` is enabled.

**Comment by Flipez** [comment]
I was able to run tf and apply the changes to the bucket without reimporting a resource. If I change behavior as requested by you it will apply what looks like a default config like so to every project

```
  # module.prod_project["whattrainisitnow"].google_logging_project_bucket_config.project will be created
  + resource "google_logging_project_bucket_config" "project" {
      + bucket_id        = "_Default"
      + description      = (known after apply)
      + enable_analytics = false
      + id               = (known after apply)
      + lifecycle_state  = (known after apply)
      + location         = "global"
      + name             = (known after apply)
      + project          = "moz-fx-whattrainisitnow-prod"
      + retention_days   = 30
    }
```

Is that what you mean?

**Comment by jasonthomas** [comment]
Yes, that's what I mean. Thanks for testing.

File: google_project/main.tf [hunk 1]
```diff
@@ -60,3 +60,11 @@ AND NOT protoPayload.serviceName="iam.googleapis.com"
 AND NOT protoPayload.serviceName="sts.googleapis.com"
   EOT
 }
+
+resource "google_logging_project_bucket_config" "project" {
+  count            = var.log_analytics ? 1 : 0
```
Comment (jasonthomas) [inline google_project/main.tf:65]: Per the previous comment, let's remove this so we define this resource for all projects.

**Review by jasonthomas — CHANGES_REQUESTED** [review]

File: google_project/main.tf (hunk 1 above)
Comment (jasonthomas) [inline google_project/main.tf:65]:
```suggestion

```

**Review by jasonthomas — APPROVED** [review]

**Pull request merged**